examples/tickets.py

```diff
@@ -0,0 +1,30 @@
+# Run this script using: python examples/tickets.py <YOUR-TOKEN>
+import sys
+from exonetapi import Client
+
+# Create a new Client.
+client = Client('https://api.exonet.nl')
+
+# Authorize with a personal access token.
+client.authenticator.set_token(sys.argv[1])
+
+print('\nID and subject of all tickets (limit 5):\n')
+# Get 5 tickets.
+all_tickets = client.resource('tickets').size(5).get()
```

Comment:
Personal preference: change `all_tickets` to `tickets`; you're not fetching all the tickets, but only a subset.